pandas/tests/series/test_reductions.py

```diff
@@ -9,6 +9,17 @@
 import pandas._testing as tm
 
 
+@pytest.mark.parametrize("operation", ["min", "max"])
+def test_reductions_series_strings(operation):
+    # GH#31746
+    list_str = ["a", "b"]
+    ser = Series(list_str, dtype="string")
+    res_operation_serie = getattr(ser, operation)()
+    func_builtin_python = __builtins__[operation]
```

Comment:
Hmm instead of this could you hardcore the expected (a or b) with the parameterize as well?